### DISCUSSION TITLE
Import view from segment_lidar fails

### DISCUSSION THREAD

**Discussion by felipenorconsult** [Yarroudh/segment-lidar#18] [Q&A]
ImportError: cannot import name 'view' from 'segment_lidar' (C:\Python\Python311\Lib\site-packages\segment_lidar\__init__.py)

**Comment by Yarroudh** [comment]
This is not the correct file C:\Python\Python311\Lib\site-packages\segment_lidar_init_.py
Did you check your configuration?

**Reply by felipenorconsult** [reply]
Hi
I just installed and run the script. 
![image](https://github.com/Yarroudh/segment-lidar/assets/123383337/46cb0fe3-f3fe-42f4-80c4-2a5adb5d4f4f)

**Reply by felipenorconsult** [reply]
windows 10
